sdk/ml/azure-ai-ml/azure/ai/ml/dsl/_parallel_for.py

```diff
@@ -0,0 +1,48 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+from azure.ai.ml.entities._builders.parallel_for import ParallelFor
+
+
+def parallel_for(*, body, items):
```

Comment:
shall we allow kwargs here? 
would you remind me what's the design reason we don't allow max_concurrency here?

Reply:
Not sure if backend will support this then. Shall we hide this in kwargs since it's not supported currently?

Reply:
Supported max_concurrency in kwargs